engine/src/flutter/display_list/dl_op_flags.h

```diff
@@ -323,6 +323,12 @@ class DisplayListOpFlags : DisplayListFlags {
       kBasePaintFlags |       //
       kBaseStrokeOrFillFlags  //
   };
+  static constexpr DisplayListAttributeFlags kDrawRSuperellipseFlags{
+      kBasePaintFlags |  //
+      kIsFilledGeometry  //
+                         // TODO(dkwingsmt): Support strokes.
```

Comment:
also kUsesMaskFilter and kUsesAntialias. Everything in kBaseStrokeFlags except for IsDrawn... (We do need mask filters on these for shadows, don't we? And AA is ignored by Impeller, but it matters for Skia so we need to make sure we list that flag so we synch that attribute.)

Reply:
Since I've added support for stroking RSEs, I've changed the flags to be the same as RRects. Can you take a look if it's correct?